inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -733,13 +893,12 @@ int main(int argc, char* argv[]) {
                 }
                 statistics->addParameters(StatisticsReport::Category::EXECUTION_RESULTS,
                                           {
-                                              {latency_label, double_to_string(latency)},
+                                              {latency_label, double_to_string(medianLatency)},
```

Comment:
There is a sense to add new latencies to the report

Reply:
Added https://github.com/openvinotoolkit/openvino/pull/7788/commits/d5a2e78f5adbd4978ce0bf73737405c9c7eed3b4